crowdsec-docs/unversioned/user_guides/plugins/whm.md

```diff
@@ -0,0 +1,152 @@
+---
+id: whm
+title: WHM plugin
+sidebar_position: 10
+---
+
+The WHM plugin offers a streamlined approach for controlling and monitoring the CrowdSec Security Engine protecting your server directly from your WHM dashboard.  
+Designed specifically for WHM environments, this plugin comes packaged with configurations tailored to enhance your server's security posture efficiently.  
+
+In a few easy steps, this guide walks you through the installation of the WHM plugin, and post-install verification that everything is working fine.
+
+## Installation
+
+### Prerequisites: install CrowdSec
+
+To be able to use this plugin, the first step is to [install CrowdSec Security Engine](https://doc.crowdsec.net/docs/getting_started/install_crowdsec) on your WHM server.
+
+### Download the plugin
+
+First, connect to your WHM server.
+
+Go to your home directory or any directory that can be used to download the sources.
+
+Choose [the release X.Y.Z you want to install](https://github.com/crowdsecurity/cs-whm-plugin/releases),
+and:
+
+* Download the source code archive.
+
+
+```shell
+wget https://github.com/crowdsecurity/cs-whm-plugin/archive/refs/tags/vX.Y.Z.tar.gz
+```
+
+* Extract sources:
+
+```shell
+tar -xvf vX.Y.Z.tar.gz
+``` 
+
+* Go to the extracted folder:
+
+```shell
+cd cs-whm-plugin-X.Y.Z/plugin
```

Comment:
Make this as simple as `cd cs-whm-plugin-*`

Reply:
might be confusing if he has multiple versions in his folder. I agree that if he does he's probably clever enough to know not to use wildcard but directly target the folder he want

Reply:
i'll change it in next update if necessary

Reply:
Or you risk users opening issues saying `cd cs-whm-plugin-X.Y.Z/plugin` says there no directory (I know its extreme but we had users like it)

Reply:
done